### PR TITLE
Docs: Ingress routing order

### PR DESCRIPTION
**Pull request by alexfornuto**
## Summary

Documents the order in which routes are applied when using the Ingress Controller.

## Related issues

Documents https://github.com/pomerium/ingress-controller/pull/145

## Checklist

- [x] reference any related issues
- [x] updated docs
- [ ] updated unit tests
- [ ] updated UPGRADING.md
- [x] add appropriate tag (`improvement` / `bug` / etc)
- [x] ready for review

### CODE REVIEW

**Review by calebdoxsey — APPROVED** [review]

File: docs/docs/k8s/ingress.md [hunk 1]
```diff
@@ -124,6 +124,18 @@ metadata:
...
 
+::: tip
+Routes are sorted and applied in the following order:
+
+- ascending by `from`,
+- descending by `path`,
+- descending by `regex`,
+- descending by `prefix`,
+- ascending by `id`.
+
```
Comment (cmo-pomerium) [inline docs/docs/k8s/ingress.md:135]: Overall LGTM. Some minor notes:

- It took me a bit of time to understand in context why this section exists here when looking at the preview.
- I don't understand why you arranged it as Ascending, then 3 Descending, then Ascending again. I don't feel strongly about it but perhaps put both Ascending together? If there's a specific reason disregard.

**Review by cmo-pomerium — APPROVED** [review]

File: docs/docs/k8s/ingress.md (hunk 1 above)
Comment (alexfornuto) [reply docs/docs/k8s/ingress.md:135]: @cmo-pomerium that's the order in which they are processed. Perhaps I should make it a numbered list, so it's more clear that it's ordered?

File: docs/docs/k8s/ingress.md [hunk 2]
```diff
@@ -124,6 +124,18 @@ metadata:
...
 :::
 
+::: tip
+Routes are sorted and applied in the following order:
+
+- ascending by `from`,
+- descending by `path`,
+- descending by `regex`,
+- descending by `prefix`,
+- ascending by `id`.
```
Comment (alexfornuto) [inline docs/docs/k8s/ingress.md:134]:
```suggestion
1. ascending by `from`,
1. descending by `path`,
1. descending by `regex`,
1. descending by `prefix`,
1. ascending by `id`.
```

File: docs/docs/k8s/ingress.md [hunk 3]
```diff
@@ -124,6 +124,18 @@ metadata:
...
 
+::: tip
+Routes are sorted and applied in the following order:
+
+1. ascending by `from`,
+1. descending by `path`,
+1. descending by `regex`,
+1. descending by `prefix`,
+1. ascending by `id`.
+
```
Comment (alexfornuto) [inline docs/docs/k8s/ingress.md:135]:
```suggestion
Routes are sorted and applied in the following order.

1. Ascending by `from`,
1. Descending by `path`,
1. Descending by `regex`,
1. Descending by `prefix`,
1. Ascending by `id`.

```

Copy edits to match the new ordered list.

**Pull request merged**